scripts/slurmsync.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# nstest
```

Comment:
I think this is a leftover comment from development

Reply:
Good catch, thanks and fixed.